CI_Automation/Documentations/Readme

```diff
@@ -0,0 +1,336 @@
+Dent CI Automation
```

Comment:
there is already a docs folder for documentation, have docs there so it's all in one place and link to a page that make sense

also readme file should be READ.me or whatever the github standard is